honeybee_schema/update/version_1_39_12.py

```diff
@@ -0,0 +1,19 @@
+"""Changes associated with version Honeybee schema version 1.39.12."""
+from ._base import Updater
+
+
+class Version_1_39_12(Updater):
```

Comment:
I'm not sure if you need use classes for this case. Right now you have a baseclass which doesn't do anything in particular and then subclass from that and again not using any of class features and finally add a `staticmethod` which doesn't need the class! Can't we just simply write them as functions and name them accordingly. That should do the same with less boilerplate code.

To give an analogy that might help. Your approach is similar to Python's unittest (https://docs.python.org/3/library/unittest.html#basic-example). What I'm suggesting is more like pytest which does the same thing by using functions.